src/test/java/com/answerdigital/answerking/service/OrderServiceTest.java

```diff
@@ -222,20 +239,21 @@ void testUpdateOrderWithInvalidOrderIdThrowsNotFoundException() {
     void testUpdateOrderWithAlreadyCancelledOrderThrowsOrderCancelledException() {
         // Given
         final Order order = orderTestBuilder
-                .withDefaultValues()
-                .withOrderStatus(OrderStatus.CANCELLED)
-                .build();
+            .withDefaultValues()
+            .withId(1L)
+            .withOrderStatus(OrderStatus.CANCELLED)
+            .build();
         final OrderRequest updateOrderRequest = orderRequestTestBuilder
-                .withLineItemRequests(List.of(new LineItemRequest(1L, 1)))
-                .build();
+            .withLineItemRequests(List.of(new LineItemRequest(1L, 1)))
+            .build();
 
         // When
         doReturn(Optional.of(order))
             .when(orderRepository)
             .findById(anyLong());
 
         // Then
-        assertThrows(OrderCancelledException.class, () -> orderService.updateOrder(order.getId(), updateOrderRequest));
+        assertThrows(OrderCancelledException.class, () -> orderService.updateOrder(1L, updateOrderRequest));
```

Comment:
Magic value here, can we please replace this with updateOrderRequest.getId()

Reply:
Ah yes, apologies! I was thinking back when I last did this Sonar was complaining with: Refactor the code of the lambda to have only one invocation possibly throwing a runtime exception.

So I temporarily used a magic value until I could find a solution, though I guess this wouldn't be necessary as we know at the point of execution that this _will never be thrown_.